app/presenters/failure_presenter.rb

```diff
@@ -4,17 +4,14 @@ class FailurePresenter
   STATE_CONFIG = {
     failure: {
       icon: 'alert/fail-x.svg',
-      alt_text: 'failure',
```

Comment:
It looked like these were intended to be used by https://github.com/18F/identity-idp/pull/4000/files#diff-122cc7e55f2d80a50ced80cba768e4df, but were not. The images are decorative so I added an empty alt tag instead.